ini-file-parser/src/test/java/com/baeldung/inifileparser/TestIniFileParser.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.inifileparser;
+
+import java.io.IOException;
+
+import org.ini4j.InvalidFileFormatException;
+import org.junit.Test;
+
+public class TestIniFileParser {
+
+    @Test
+    public void testIni4jParser() throws InvalidFileFormatException, IOException {
```

Comment:
Method doesn't need to start `test` we know it's a test. Use the `given`/`when`/`then` syntax. Also use `throws Exception` as we don't care what precise exception this throws.